packages/flutter/lib/src/widgets/navigation_toolbar.dart

```diff
@@ -76,17 +77,20 @@ enum _ToolbarSlot {
 
 const double _kMiddleMargin = 16.0;
 
-// TODO(xster): support RTL.
 class _ToolbarLayout extends MultiChildLayoutDelegate {
-  _ToolbarLayout({ this.centerMiddle });
+  _ToolbarLayout({
+    this.centerMiddle,
+    this.textDirection,
```

Comment:
should this deal with null somehow?

Reply:
We generally don't have a default `TextDirection` because that would introduce either an LTR or RTL bias.  I'll add an assert.

Reply:
Done.